context/src/test/java/io/opentelemetry/context/propagation/internal/ExtendedTextMapGetterTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.context.propagation.internal;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import javax.annotation.Nullable;
+import org.junit.jupiter.api.Test;
+
+public class ExtendedTextMapGetterTest {
```

Comment:
```suggestion
class ExtendedTextMapGetterTest {
```

Reply:
Won't we need this to be public in order for instrumentation to adopt it? What's the path towards stability?